src/panel/servicePanel.jsx

```diff
@@ -0,0 +1,64 @@
+/* global _ */
+import React from 'react';
+import Panel from 'src/components/ui/Panel';
+import CategoryService from '../adapters/category_service';
+import nconf from '@qwant/nconf-getter';
+
+class ServicePanel extends React.Component {
+  render() {
+    return <Panel
+      resizable
+      title={_('Qwant Maps services', 'service panel')}
+      minimizedTitle={_('Show Qwant Maps services', 'service panel')}
+      className={'service_panel'}
+    >
+      <div className="service_panel__categories">
+        {
```

Comment:
This part could be extracted in a separate component so it's ready to be used by the POI panel. But let's wait for it to be merged first ;)